.github/workflows/build_docker_container.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Push Docker Container
+
+on:
+  push:
+    paths:
+      - ".devcontainer/Dockerfile"
+      - ".github/workflows/build_docker_container.yml"
+      - "requirements.txt"
+  workflow_dispatch:
+
+concurrency:
+  group: docker
+  cancel-in-progress: true
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.SSH_PRIVATE_KEY }}
```

Comment:
add docs for all secrets needed in the transformer mothership README

Reply:
Done